pkg/cmd/pipelinerun/describe.go

```diff
@@ -73,8 +99,40 @@ or
 		},
 	}
 
+	c.Flags().IntVarP(&opts.Limit, "limit", "", defaultDescribeLimit, "lists number of pipelineruns")
+	c.Flags().BoolVarP(&opts.Fzf, "fzf", "F", false, "use fzf to select a pipelinerun")
+
 	_ = c.MarkZshCompPositionalArgumentCustom(1, "__tkn_get_pipelinerun")
 	f.AddFlags(c)
 
 	return c
 }
+
+func askPipelineRunName(opts *options.DescribeOptions, p cli.Params) error {
+	lOpts := metav1.ListOptions{}
+
+	err := opts.ValidateOpts()
+	if err != nil {
+		return err
+	}
+
+	prs, err := prhelper.GetAllPipelineRuns(opts.Params, lOpts, opts.Limit)
+
+	if err != nil {
+		return err
+	}
+	if len(prs) == 0 {
+		return fmt.Errorf("No pipelineruns found")
```

Comment:
Always use [lowercase for beginning an error message](https://github.com/golang/go/wiki/CodeReviewComments#error-strings). We need to be better about enforcing this in the CLI moving forward, so apologies if you saw this elsewhere in the CLI.

```suggestion
		return fmt.Errorf("no pipelineruns found")
```

Reply:
Sure, will keep that in mind. made the change.